### DISCUSSION TITLE
Some questions regarding Lokinet

### DISCUSSION THREAD

**Discussion by Raj2032** [oxen-io/lokinet#1889] [Q&A] [answered]
Hi I have some questions regarding this project:

1. Since Lokinet does not use centralised service like Tor browser does, if I am using lokinet, then are other people using my network to send and receive packets?
2. Am I able to access `.onion` websites like the tor browser since this is an onion router?

**Comment by majestrate** [comment, marked as answer]
1. No; traffic is carried across the network by Oxen Service Nodes (see https://oxen.io).

2. No. Tor and lokinet have no connection. Lokinet *does* allow clients to talk to each other via .loki addresses, but there is no commonality between the Tor and the Lokinet networks.
